src/lib/convert.ts

```diff
@@ -112,11 +114,22 @@ const getVariableName = (convertionOptions: ConvertionOptions, filenameWithoutEn
   return `${convertionOptions.prefix}${capitalize(camelCase(filenameWithoutEnding))}`;
 };
 
-const extractSvgContent = async (fileName: string, directoryPath: string): Promise<string> => {
-  const fileContentRaw = await readfile(path.join(directoryPath, fileName), 'utf-8');
+const extractSvgContent = async (filePath: string): Promise<string> => {
+  const fileContentRaw = await readfile(filePath, 'utf-8');
   return fileContentRaw.replace(/\r?\n|\r/g, ' ');
 };
 
 const capitalize = (value: string): string => {
   return value.charAt(0).toUpperCase() + value.slice(1);
 };
+
+const toRegexString = (filesRegex: string): string => {
```

Comment:
Yes, I see the issue. Maybe it's better if we add a new argument next to the srcDirectory. Another thing could be that we decide to only support regex for srcDirectory. Downside, this would mean that we would bump the version and introduce a breaking change.

I think if it helps us to keep the code clean then I am not against the breaking change. Since we are using semver it should not be a problem. Otherwhise if we can easily mix the two concepts (regex and non regex) we can provide an additional argument for the regex.

Reply:
The decision is all yours.

But I think we are both not satisfied with this first solution I've made.

Reply:
I think it would be the best to change the behaviour of srcDirectory to only support regex. This would result in a breaking change. I think its better to do it with a breaking change than with an additional flag. It's easier to keep our internals clean.